devicon.json

```diff
@@ -928,8 +1009,27 @@
         "color": "#0e8a16",
         "aliases": []
     },
+    {
+        "name": "clojurescript",
+        "altnames": [],
+        "tags": [
+            "language"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "plain"
+            ],
+            "font": [
+                "plain"
+            ]
+        },
+        "color": "#96ca4b",
+        "aliases": []
+    },
```

Comment:
The same applies for `clojurescript`, already added.

Reply:
https://github.com/devicons/devicon/pull/1167#discussion_r873515186